docs/content/preview/tutorials/AI/ai-localai.md

```diff
@@ -17,11 +17,47 @@ This tutorial shows how you can use [LocalAI](https://localai.io/) to create an
 
 ## Prerequisites
 
-* YugabyteDB [v2.19.2 or later](https://download.yugabyte.com/)
+* YugabyteDB [v2.25+](https://download.yugabyte.com/)
 * [LocalAI](https://localai.io/basics/getting_started/)
 * Python 3
 * Docker
 
+## Set up YugabyteDB
+
+Start a 3-node YugabyteDB cluster in Docker (or feel free to use another deployment option):
+
+```sh
+# NOTE: if the ~/yb_docker_data already exists on your machine, delete and re-create it
+mkdir ~/yb_docker_data
+
+docker network create custom-network
+
+docker run -d --name yugabytedb-node1 --hostname yugabytedb-node1 --net custom-network \
```

Comment:
@HarshDaryani896 you made modifications to this docker command for another tutorial - do they need to be made here? Which version should be standard for the tutorial Docker setup